src/components/measuretool/MeasureResult.vue

```diff
@@ -35,6 +35,11 @@ export default {
       if (geom instanceof PolygonGeom) {
         output = me.formatArea(geom);
         me.area = output;
+
+        // perimeter of  outer LinearRing of measure polygon
```

Comment:
duplicate whitespace

```suggestion
        // perimeter of outer LinearRing of measure polygon
```